Tensile/Tests/extended/stagger_u/big_skinny_A_NN.yaml

```diff
@@ -28,16 +28,6 @@ BenchmarkProblems:
 
     - # BenchmarkProblemSizeGroup - Standard
       InitialSolutionParameters:
```

Comment:
Do the section headers like this one have to stay in if the params are no longer supported?

Reply:
Hmm, I suppose I did it this way for consistency with the other yaml test configs. Or at least, that's the best reason I can think of for why I didn't just outright remove them.

To answer the question though, no, the section headers are not required. All the other yamls should be updated as well though, and I think there might be some other quirks that should be fixed as well. I'll do that as a separate PR.